tests/nexus_api_test.go

```diff
@@ -363,6 +423,8 @@ func (s *ClientFunctionalSuite) TestNexusStartOperation_Forbidden() {
 func (s *ClientFunctionalSuite) TestNexusStartOperation_Claims() {
 	taskQueue := s.randomizeStr("task-queue")
 	testService := s.createNexusIncomingService(s.randomizeStr("test-service"), taskQueue)
+	// Wait to make sure incoming service is loaded into memory before starting tests.
+	time.Sleep(3 * time.Second)
```

Comment:
This is always a smell, you can use `EventuallyWithT` and assert that the service is ready.
Can do it in the next PR though.